python/lsst/obs/decam/decamMapper.py

```diff
@@ -233,7 +241,11 @@ def std_raw(self, item, dataId):
 
     def std_dark(self, item, dataId):
         exp = afwImage.makeExposure(afwImage.makeMaskedImage(item))
-        exp.getCalib().setExptime(1.0)
+        rawPath = self.map_raw(dataId).getLocations()[0]
```

Comment:
I suppose it is possible, but I think it unlikely. It should not happen during ISR.